plugins/outputs/mqtt/mqtt_v3.go

```diff
@@ -0,0 +1,92 @@
+package mqtt
+
+import (
+	"fmt"
+	"time"
+
+	// Library that supports v3.1.1
+	mqttv3 "github.com/eclipse/paho.mqtt.golang"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/internal"
+)
+
+type MQTTv311 struct {
+	*MQTT
+	client mqttv3.Client
+}
```

Comment:
Do we need to export this structure or can we leave it unexported?
```suggestion
type mqttv311 struct {
	*MQTT
	client mqttv3.Client
}
```